src/main/java/org/fcrepo/importexport/ArgParser.java

```diff
@@ -137,6 +137,13 @@
                 .desc("When present this flag indicates that inbound references should be exported.")
                 .required(false).build());
 
+        // Omit membership
+        configOptions.addOption(Option.builder()
+                .longOpt("omit-membership")
+                .hasArg(false)
+                .desc("When present this flag indicates that membership references should NOT be exported.")
```

Comment:
I believe we have a choice here. Since having 'omit-membership' is required for migration to F6, I would suggest we either:
- change the default to `true`, or
- add more usage text (and README text) indicating that the option has to be set to `true` for migration.

Reply:
I'm happy with either option. @dbernstein do you have an opinion?

Reply:
I am leaning a little towards changing the default to `true`.
￼I believe that is the common-case for this tool.